openpype/hosts/maya/plugins/publish/validate_rendersettings.py

```diff
@@ -121,7 +121,8 @@ def get_invalid(cls, instance):
             cls.log.error("Animation needs to be enabled. Use the same "
                           "frame for start and end to render single frame")
 
-        if not prefix.lower().startswith("maya/<scene>"):
+        if not prefix.lower().startswith("maya/<scene>") and \
+                renderer != "renderman":
```

Comment:
Shouldn't we still consider it invalid for Renderman too if it doesn't write into `maya/scenes`?

Should it be something like this?

**Consider this untested pseudocode**
```suggestion
        required_prefix = "maya/<scene>"
        if renderer == "renderman":
            # Renderman requires to start with <ws> to write into the workspace
            required_prefix = "<ws>/" + required_prefix

        if not prefix.lower().startswith(required_prefix):
```

Reply:
Also looking at the [Renderman Documentation](https://rmanwiki.pixar.com/display/RFM22/Workspace) shouldn't this prefix actually also contain [the render folder from the actual workspace](https://github.com/pypeclub/OpenPype/blob/develop/openpype/hosts/maya/resources/workspace.mel#L11)?

Something like:

`<ws>/renders/maya/<scene>`?